lib/pages/home.dart

```diff
@@ -88,20 +88,26 @@ class HomePageState extends State<HomePage>
   }
 
   Widget _buildAppBar(BuildContext context) {
+    final isPortrait = MediaQuery.of(context).orientation == Orientation.portrait;
+
     return AppBar(
       bottom: PreferredSize(
         child: Container(),
-        preferredSize: Size.fromHeight(45.0),
+        preferredSize: isPortrait
+          ? Size.fromHeight(45.0)
+          : Size.fromHeight(0.0),
       ),
-      flexibleSpace: _buildFormFields(context),
+      flexibleSpace: isPortrait
+        ? _buildFormFieldsPortrait(context)
+        : _buildFormFieldsLandscape(context),
     );
   }
 
-  Widget _buildFormFields(BuildContext context) {
+  Widget _buildFormFieldsPortrait(BuildContext context) {
     final localizations = TrufiLocalizations.of(context);
     return SafeArea(
       child: Container(
-        padding: EdgeInsets.all(4.0),
+        padding: EdgeInsets.fromLTRB(12.0, 4.0, 4.0, 4.0),
```

Comment:
I noticed the spacing to the menu icon is off. This gives it proper spacing.

Reply:
nice!